detox/android/detox/src/main/java/com/wix/detox/DetoxManager.java

```diff
@@ -145,5 +145,9 @@ public Unit invoke() {
                 return null;
             }
         }));
+
+        final DetoxInstrumentsManager instrumentsManager = new DetoxInstrumentsManager(reactNativeHostHolder);
+        actionHandlers.put("setRecordingState", new RecordingStateActionHandler(instrumentsManager, wsClient));
+        actionHandlers.put("addEvent", new EventsActionsHandler(instrumentsManager, wsClient));
```

Comment:
Could we change this so these will only be added conditionally (i.e. if indeed instrumentation exists)?